src/app/dashboard/prompt-gallery/page.tsx

```
@@ -86,6 +149,34 @@ export default function PromptGalleryPage() {
     // You could add a toast notification here
   };
 
+  const handleUseTemplate = async (template: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)